### DISCUSSION TITLE
Commercial Support

### DISCUSSION THREAD

**Discussion by akibsayyed** [baresip/baresip#1938] [General]
Hi 
I am looking to create some custom Out of box modifications in the baresip. I would like to know if you can provide commercial support for same.

**Comment by akibsayyed** [comment]
@sreimers 
Thank you 

We can pay BareSIP Team for our custom modification requirements. Will you be interested to provide support?

**Comment by alexlansovich** [comment]
Hello.
I also have this question. can you write me proposal?

**Comment by juha-h** [comment]
Might help if you tell what you need to be done.

**Reply by alexlansovich** [reply]
I need support for a integrating bare sip into intercom system.
can you write me at alexlans@gmail.com ?
i will write all information.
